lib/screens/profile_onboarding.dart

```diff
@@ -162,26 +166,56 @@ class _ProfileOnboardingState extends State<ProfileOnboarding> {
     );
   }
 
-  void _validateAndSubmit() {
+  void _validateAndSubmit() async {
     if (!_formKey.currentState.validate()) {
       return;
     }
     _formKey.currentState.save();
 
-    _birthDate = DateTime.parse('$_birthYear-$_birthMonth-$_birthDay');
+    _birthDate = DateTime.parse(
+        '$_birthYear-$_birthMonth-$_birthDay'); // we're not even using this
+    User _firebaseUser = EmailAuth().getCurrentUser();
+    ProfileOnboardingService _profileOnboardingService =
+        new ProfileOnboardingService();
 
-    print('$_role, $_firstName, $_lastName, $_city, $_birthDate');
+    /// Create the user object to be sent out.
+    UserModel user = new UserModel(
+      id: _firebaseUser.uid,
+      firstName: _firstName,
+      lastName: _lastName,
+      firebaseUuid: _firebaseUser.uid,
+      email: _firebaseUser.email,
+      city: _city,
+      category: [], //we don't have form field for this
+      pictureUrl: '',
+      role: _role,
+      ratings: [],
+      averageRating: 0.0,
+    );
```

Comment:
For create profile. The Months and Day and Year do not indicate what this information is for.

Is it my birthday or the day I joined this app?
![image](https://user-images.githubusercontent.com/38572314/112520973-383c1700-8d59-11eb-9f75-4f5e2cb47c17.png)

Reply:
I am currently considering removing that field altogether. We don't have a field for it in our back-end, and our front end has no use for it to be honest. I will discuss this during the next meeting.